source/ed/docker_tests/ed_integration_tests.cpp

```diff
@@ -446,7 +446,7 @@ BOOST_FIXTURE_TEST_CASE(ntfs_dst_test, ArgsFixture) {
     data.build_raptor();
     BOOST_REQUIRE_EQUAL(failed, false);
 
-    BOOST_CHECK_EQUAL(data.pt_data->lines.size(), 1);
+    BOOST_CHECK_EQUAL(data.pt_data->lines.size(), 2);
```

Comment:
because of [updated lines.txt](https://github.com/CanalTP/navitia/pull/2911/files#diff-30567d06e400cc530b104de208e9cdc5R3)